harmonica/normal_gravity.py

```diff
@@ -0,0 +1,75 @@
+"""
+Gravity of ellipsoids models.
+"""
+import numpy as np
+from .ellipsoid import get_ellipsoid
+
+
+def normal_gravity(latitude, height):
+    """
+    Calculate normal gravity at any latitude and height.
+
+    Computes the magnitude of the gradient of the gravitational potential
+    generated by a reference ellipsoid on a point at a certain ``latitude`` and
+    ``height``.
+    It makes use of the closed form expression of [LiGotze2001]_ and the
+    current reference ellipsoid defined by :func:`harmonica.set_ellipsoid`.
+
+    Parameters
+    ----------
+    latitude : float or numpy array
+        The latitude where the normal gravity will be computed (in degrees)
+    height : float or numpy array
+        The height of computation (in meters). Should be ellipsoidal
+        (geometric) heights for geophysical purposes.
+
+    Returns
+    -------
+    gamma : float or numpy array
+        The computed normal gravity (in mGal).
+
+    """
+    ellipsoid = get_ellipsoid()
+    ecc = ellipsoid.linear_eccentricity
+    omega2 = ellipsoid.angular_velocity**2
+
+    big_w, q_l, q_0, b_l, cosbeta_l2, sinbeta_l2 = _normal_grav_factors(
+        ellipsoid, latitude, height)
+    term1 = ellipsoid.geocentric_grav_const / (b_l**2 + ecc**2)
+    term2 = 0.5 * sinbeta_l2 - 1 / 6
+    term2 *= ellipsoid.semimajor_axis**2 * ecc * q_l * omega2 / \
+        ((b_l**2 + ecc**2) * q_0)
+    term3 = -cosbeta_l2 * b_l * omega2
+    gamma = (term1 + term2 + term3) / big_w
+    # Convert gamma from SI to mGal
+    gamma *= 1e5
+    return gamma
```

Comment:
Yes, mainly because each quantity has no physical meaning (or it's not explicit on the Li and Gotze (2001) paper). I tried to write the function in a way that it could be easily readable having the paper next to the code:
* gamma can be computed as the sum of 3 terms divided by W.
* For these terms, only E, q', q_0, b' and beta' are needed.
* The geometric quantites on the second column of page 1668 (Li and Gotze (2001)) are only needed for computing the quantities of the last item.

Glad you like it!